src/pages/iou/IOURequestPage.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+import IOUModal from './IOUModal';
+
+export default props => (
+    // eslint-disable-next-line react/jsx-props-no-spreading
+    <IOUModal {...props} />
```

Comment:
How is this different than just using `IOUModal` directly? Like, it seems like this file could also just be:

```
export default IOUModal;
```

and in that case, it seems like this file isn't necessary.